apps/docs/stories/Accordian.stories.tsx

```diff
@@ -35,15 +33,15 @@ const commonArgs = {
       body: 'Accordian body value 1',
     },
   ],
-  // @ts-ignore
-  renderBody: ({ body, strongText }) => {
+  renderBody: ({ body = '', strongText = '' }: Record<string, unknown>) => {
```

Comment:
i guess this is temp, but ideally body could be another component as well right?

Reply:
renderBody first argument is item, it is the same item passed to the component. so whatever we will pass in item we will receive here.

Reply:
hmm but it accepts only string as per the Record type??

Reply:
string is the key value is unknown, we can add value of any type to body